src/memex/views.py

```diff
@@ -65,12 +66,28 @@ def __init__(self):
         )
 
 
-def api_config(**settings):
-    """
-    A view configuration decorator with defaults.
+def add_api_view(config, view, link_name=None, description=None, **settings):
```

Comment:
These configuration functions are getting a bit big, it might be time to move them out of the views module into their own. But that can happen in a second step as well.